container/container.go

```diff
@@ -234,10 +235,31 @@ func getHostname(taskInfo *mesos.TaskInfo) string {
 	return ""
 }
 
+// Map Task Env to Docker Env
+func AppendTaskEnv(envVars [] string, taskInfo * mesos.TaskInfo) [] string {
+	if taskInfo.Executor == nil || taskInfo.Executor.Command == nil ||
+		taskInfo.Executor.Command.Environment == nil ||
+		taskInfo.Executor.Command.Environment.Variables == nil {
+		return envVars
+	}
+
+	for _, vars := range taskInfo.Executor.Command.Environment.Variables {
```

Comment:
Not sure if it's worth but I would test if vars is nil in order to avoid nullptr deref in L.249.